### PR TITLE
fix B0003 example and update logs

### PR DESCRIPTION
**Pull request by mockersf**
# Objective

- Example in error B0003 is not failing anymore after #9822

## Solution

- Update the example code so that is always fail
- Also update logs and instructions on how to debug as it's easier now

### CODE REVIEW

File: errors/B0003.md
Comment (alice-i-cecile) [inline errors/B0003.md:41]:
```suggestion
This will panic, as the system that is executed first will despawn the entity used by the second.
```

**Review by alice-i-cecile — APPROVED** [review]
Grammar nit, but LGTM.

**Comment by alice-i-cecile** [comment]
@doonv @killercup could I get a review from one or both of you?

File: errors/B0003.md
Comment (doonv) [inline errors/B0003.md:19]: I think this would be better as 2 tuple arguments instead of an array.
```suggestion
struct MyEntities(Entity, Entity);
```

File: errors/B0003.md
Comment (doonv) [inline errors/B0003.md:43]: This grammar doesn't make sense.
```suggestion
The default panic message is telling you the entity (`2v0`) doesn't exist, and the command that failed. (adding a component `Hello`) and the system from which it originated (`use_1_and_despawn_0`):
```
Additionally there are 2 cases for what entity doesn't exist (either `1v0` or `2v0` based on system ordering)

File: errors/B0003.md
Comment (doonv) [inline errors/B0003.md:52]: Maybe mention that you can also configure `LogPlugin` to show debug logs?

**Review by doonv — APPROVED** [review]

**Comment by alice-i-cecile** [comment]
@mockersf go ahead and merge this once CI is green.

File: errors/B0003.md
Comment (mockersf) [reply errors/B0003.md]: using a tuple means accessing the entities from a resource with `my_entities.0 .0` which is less pretty than the array...

Comment (doonv) [reply errors/B0003.md:19]: If you do:
```rs
struct MyEntities(Entity, Entity);
```
instead of:
```rs
struct MyEntities((Entity, Entity));
```
then you can do `my_entities.0` / `my_entities.1` instead of `my_entities.0.0` or `my_entities.0[0]`

Comment (mockersf) [reply errors/B0003.md:19]: ha right 😄

**Review by doonv — APPROVED** [review]

**Pull request merged**
